src/wallet/rpcwallet.cpp

```diff
@@ -2394,13 +1949,9 @@ UniValue listtransactions(const JSONRPCRequest& request)
             "  {\n"
             "    \"address\":\"pivxaddress\",    (string) The pivx address of the transaction. Not present for \n"
             "                                                move transactions (category = move).\n"
-            "    \"category\":\"send|receive|move\", (string) The transaction category. 'move' is a local (off blockchain)\n"
-            "                                                transaction between accounts, and not associated with an address,\n"
-            "                                                transaction id or block. 'send' and 'receive' transactions are \n"
-            "                                                associated with an address, transaction id and block details\n"
-            "    \"amount\": x.xxx,          (numeric) The amount in PIV. This is negative for the 'send' category, and for the\n"
-            "                                         'move' category for moves outbound. It is positive for the 'receive' category,\n"
-            "                                         and for the 'move' category for inbound funds.\n"
+            "    \"category\":\"send|receive\",  (string) The transaction category.\n"
```

Comment:
There are actually other possible categories (`orphan`, `immature`, `generate`).
We, also, probably want to remove the mention to the `move` category in the previous field (address), now that "accounting entries" are a thing of the past.